### ISSUE TITLE
MFXTreeView no chevron icon displayed.

### ISSUE THREAD

**Issue by infinite-dev22** [palexdev/MaterialFX#326]
## Discription
MFXTreeView doesn't show the Chevron icon and throws error: java.lang.IllegalArgumentException: Icon description 'mfx-chevron-right' is invalid!

![Screenshot from 2023-06-23 12-17-15](https://github.com/palexdev/MaterialFX/assets/77667007/728109da-a2f7-44af-9dcf-62c8a6b729f1)

##MRP
```
MFXTreeView<String> tree = new MFXTreeView<>();
MFXTreeItem<String> root = new MFXTreeItem<>("");

for (int i=0; i<=4; i++) {
    MFXTreeItem<String> item = new MFXTreeItem<>("Tree Item: "+i);
    for (int j=0; j<=4; j++) {
        MFXTreeItem<String> subItem = new MFXTreeItem<>("Tree SubItem: "+j);
        item.getItems().add(subItem);
    }
    root.getItems().add(item);
}

root.setTreeView(tree);
tree.setShowRoot(false);
tree.setRoot(root);
```

**Comment by bcrdaniel** [comment]
Im getting the same error when update materialfx on my project

**Comment by palexdev** [comment]
Should be fixed in version `11.17.0`
